tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_model.py

```diff
@@ -166,8 +166,12 @@ def _fit(self, X_train, y_train, X_val=None, y_val=None, time_limit=None, num_cp
         start_time = time.time()
         try_import_mxnet()
         import mxnet as mx
-        params = self.params.copy()
         self.verbosity = kwargs.get('verbosity', 2)
+        weights = kwargs.get('weights', None)
```

Comment:
as a future follow-up, how much work would it be to use weights in NN models?

Reply:
added TODOs for neural net models. Not sure how hard it will be for fastAI neural network. Is relatively straightforward for MXNet neural network in future PR, but will likely require extensive benchmarking to catch edge cases.